scripts/qa_crowbarsetup.sh

```diff
@@ -2019,21 +2018,22 @@ function reboot_nodes_via_ipmi()
             if [ $i -gt $nodenumber ]; then
                 # power off extra nodes
                 ipmitool -H $ip -U root -P $pw power off &
+                wait_for 2 60 "ipmitool -H $ip -U root -P $pw power status | grep -q 'is off'" "node to power off"
             else
                 ping -c 3 $ip > /dev/null || {
                     echo "error: BMC $ip is not reachable!"
                 }
                 (
                 ipmitool -H $ip -U root -P $pw lan set 1 defgw ipaddr "${bmc_values[1]}"
-                sleep $((5 + RANDOM % 5))
+                sleep $((50 + RANDOM % 20))
 
                 if ipmitool -H $ip -U root -P $pw power status | grep -q "is off"; then
                     ipmitool -H $ip -U root -P $pw power on
-                    sleep $((10 + RANDOM % 15))
+                    sleep $((50 + RANDOM % 25))
```

Comment:
This could also be: wait_for 2 60 "ipmitool -H $ip -U root -P $pw power status | grep -q 'is on'" "waiting for node to power on"